docs/src/user/variables-guide.rst

```diff
@@ -116,6 +116,17 @@ The variables to be configured are::
 Usually, ‘m1.medium’ will suffice this requirement, but again, it can
 be different between deployments.
 
+Conversion hosts specific floating IPs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each conversion host needs to have a floating IP,
+this floating IP can be assigned automatically or
+defined by the operator with the usage of the
+following variables::
+
+    os_migrate_src_conversion_fixed_floating_ip_address    
+    os_migrate_dst_conversion_fixed_floating_ip_address
+
```

Comment:
I think the naming pattern should be just `os_migrate_src_conversion_floating_ip_address` (and dst respectively). The term "fixed IP address" is a differently defined term in OpenStack, so we should avoid confusing users here.

Reply:
done